components/bl-double-calendar-component/src/date-range.js

```diff
@@ -2,6 +2,14 @@ import { useState, useEffect } from 'react';
 
 import DatePicker from './lib/react-datepicker.min.js';
 
+const { cn } = BackendlessUI.CSSUtils;
+
+// this is needed to display the current month in the calendar after the reset without selecting the current day
+const setCurrentMonth = setState => {
+  setState(new Date());
+  setTimeout(() => setState(null), 1);
+};
```

Comment:
I don't think it is a good practice when we create a setter function which will accept another setter. Than not looks obvious.  Will be better to create handlers using useCallback inside the component.